openedx/core/djangoapps/user_authn/views/register.py

```diff
@@ -492,6 +492,7 @@ def get(self, request):
                             content_type="application/json")
 
     @method_decorator(csrf_exempt)
+    @method_decorator(ratelimit(key=REAL_IP_KEY, rate=settings.REGISTRATION_RATELIMIT, method='POST', block=True))
```

Comment:
Instead of blocking at the decorator level could handle it same as we are doing in the login endpoint? Since it's an API endpoint and `django-ratelimit` returns an HTML response with an empty body by default upon blocking. Code reference https://github.com/edx/edx-platform/blob/c260f72c2e50f85dbfef03a3ecd42a26bc3a6ea4/openedx/core/djangoapps/user_authn/views/login.py#L185-L187

Reply:
And send an error code to handle it on MFE.

Reply:
I will look into it and update PRs